libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -39,6 +39,7 @@ import { buildCastVoteRecordReportMetadata } from './build_report_metadata';
 export interface ResultSheet {
   readonly id: Id;
   readonly batchId: Id;
+  readonly batchSequenceId?: number; // central scanner only
```

Comment:
Why central scanner only? Can you add some more context to this comment?

Reply:
It would compromise voter privacy in a precinct scanner, can add more of a comment.